### ISSUE TITLE
How can I  set  the  default fontSize

### ISSUE THREAD

**Issue by huangjiangit** [jpuri/react-draft-wysiwyg#551]
It seems that there is no API to set up

fontSize: {
              defaultSize: 18,//**This does not work.**
              options: [10,12, 14, 16, 18, 24, 30, 36, 48, 60],
            },

**Comment by jpuri** [comment]
This can be done using `editorStyle` or `editorClassName`. Simply setting css `font-size` will change font-size of the editor.

**Comment by shakdoesgithub** [comment]
Hi @jpuri your solution only changes what is displayed in the editor. Is there a way to set the default font size of the draft js content?

**Comment by jpuri** [comment]
Good point, its missed. Atm you can do that by selecting font-size from dropdown.

This default font-size information is required to be added to all the blocks, I will try to fix this.

**Comment by gnyani** [comment]
@jpuri Even I have a requirement to change the default font size. Can you let me know when you will be releasing the fix?

**Comment by jpuri** [comment]
Default font-size can be changed by using editorStyle or editorClassName. Trouble atm is that its not saved in editor-content.

A work around could set default size of font while displaying editor content next time also.

**Comment by emipc** [comment]
I'm facing the same issue. Would be great to have a default fontSize option!

**Comment by Pratapvirendra47** [comment]
Is this fixed??

**Comment by mandarspringct** [comment]
Hi @jpuri any updates on this? I have same issue.
Setting font size through editorStyle / editorClassName only sets content font size. Does not change the default font size in the dropdown.
Hope for a fix soon,
Thanks!

**Comment by shaunjacobsen** [comment]
Hello, also wondering if this is going to be available as an option soon?

**Comment by sachinkammar** [comment]
Hello @jpuri please update the status of this issue.

**Comment by jakeleventhal** [comment]
Similarly, users need to be able to set the default fontFamily. Currently setting the font family will change the display, but not the actual content of the editor.

**Comment by sandropoluan** [comment]
Any update?

**Comment by sandropoluan** [comment]
I need this feature

**Comment by mikeappell** [comment]
My solution was this, until something exists in the repo's API:

```
setDefaultFontFamilyAndSize = (editorState) => {
  const currentStyles = editorState.getCurrentInlineStyle();
  let newEditorState = editorState;

  if (!currentStyles.some(style => style.match(/^fontfamily-/))) {
    newEditorState = RichUtils.toggleInlineStyle(
      newEditorState,
      `fontfamily-${DEFAULT_FONT_FAMILY}`,
    );
  }
  if (!currentStyles.some(style => style.match(/^fontfamily-/))) {
    newEditorState = RichUtils.toggleInlineStyle(
      newEditorState,
      `fontsize-${DEFAULT_FONT_SIZE}`,
    );
  }

  return newEditorState;
}
```

Make sure you call that both in your initial editorState config and anytime you process an update to editorState to make sure you're not reset back to empty font styles.

**Comment by SangeetAgarwal** [comment]
Is there a way to add a font family and keep all the other toolbar options

**Comment by ledesmablt** [comment]
It seems as though setting `editorStyle` to something like `fontSize: 16` sets the default selected font size to 16 (not just what is displayed). Tested on version 1.14.7

**Comment by abhayverma112** [comment]
I am using react-final-form
**i am making crud app every thing working fine but when i do edit with the help of this editor, first time it work perfect but when i again edit the same task it does not show the previous style in editor box**
`<Field
    component={DraftEditor}
    name="description"
    type="text"
    />`

`/* eslint-disable no-underscore-dangle */
/* eslint-disable react/destructuring-assignment */
/* eslint-disable max-len */
/* eslint-disable no-undef */
/* eslint-disable react/prop-types */
/* eslint-disable no-unused-vars */
/* eslint-disable no-shadow */
import * as React from 'react';
import { useState } from 'react';
import {
 EditorState, convertToRaw, ContentState, convertFromHTML, Modifier, editorRef,
} from 'draft-js';
import { stateFromHTML } from 'draft-js-import-html';
import { Editor } from 'react-draft-wysiwyg';
import draftToHtml from 'draftjs-to-html';
import htmlToDraft from 'html-to-draftjs';
import parser, { processNodes, convertNodeToElement, htmlparser2 } from 'react-html-parser';

const WYSIWYGEditor = ({ input, meta }) => {
  const blocksFromHTML = convertFromHTML(input.value);
  const content = ContentState.createFromBlockArray(
    blocksFromHTML.contentBlocks,
    blocksFromHTML.entityMap,
  );
  const raw = convertToRaw(content);
  const [contentState, setContentState] = useState(raw);
  const [editorState, setEditorState] = useState(EditorState.createEmpty());

  const onEditorStateChange = (contentState) => {
    setEditorState(contentState);
    return input.onChange(draftToHtml(convertToRaw(editorState.getCurrentContent())));
  };
  return (
    <div className="editor">
      <Editor
        defaultContentState={contentState}
        onEditorStateChange={onEditorStateChange}
        wrapperClassName="wrapper-class"
        editorClassName="editor-class"
        toolbarClassName="toolbar-class"
        placeholder="Write your answere here"
        toolbar={{
          options: [
            'inline',
            'blockType',
            'fontSize',
            'fontFamily',
            'colorPicker',
          ],
          inline: {
            options: ['bold', 'italic', 'underline'],
          },
          fontSize: {
            options: [8, 9, 10, 11, 12, 14, 16, 18, 24, 30, 36, 48, 60, 72, 96], 
            className: undefined,
            component: undefined,
            dropdownClassName: undefined,
          },
          blockType: {
            options: ['Normal', 'H1', 'H2', 'H3', 'H4', 'H5', 'H6', 'Blockqoute', 'Code'],
          },
          fontFamily: {
            options: ['Arial', 'Georgia', 'Impact', 'Tahoma', 'Times New Roman', 'Verdana'],
            className: undefined,
            component: undefined,
            dropdownClassName: undefined,
          },
          colorPicker: {
            className: undefined,
            component: undefined,
            popupClassName: undefined,
          },
        }}
      />
    </div>
  );
};

export default WYSIWYGEditor;
`

**Comment by gonenoob** [comment]
.DraftEditor-root {
font-size: 14px;
}

**Comment by gonenoob** [comment]
```css
.DraftEditor-root {
  font-size: 14px;
}
```

**Comment by Stinny** [comment]
@sparkmediatech Did you ever find a solution by chance? Using the Editor in read only mode as well and would like styles to match my entire application.

**Comment by meierac** [comment]
Hi @jpuri , is there already an official solution for setting the default font size that is stored in the editor content and not only in the website CSS? I want to print the editor content with my application, but if the user does not set a custom font size, the default font size is used when printing, which is tiny...

**Comment by jueser** [comment]
the same here

i found this solution:

`import {
    toggleCustomInlineStyle, getSelectionCustomInlineStyle,
} from 'draftjs-utils';
Secondly, on each render(!?) you should execute:

const fontSize = getSelectionCustomInlineStyle(editorState, ['FONTSIZE',]).FONTSIZE
if (!fontSize) {
    setEditorState(toggleCustomInlineStyle(editorState, 'fontSize', 24))
}`

but it's not wor for fontSize and fontFamily on the same time.

**Comment by maqboolna** [comment]
Adding this line set the font size 
    editorStyle={{
      fontSize: 24,
    }}



```
<ReactDraftWysiwyg
     wrapperClassName="wrapper-class"
     editorClassName="editor-class"
     toolbarClassName="toolbar-class"
    editorState={editorValue}
    editorStyle={{
      fontSize: 24,
    }}
    }}
  />
```

**Comment by Oleksandr-Korniienko-IS** [comment]
@jpuri Hi. The logic below partly works, but it only updates one property (the latest). Could you please provide a solution of how default text styles can be applied? FYI, setting it with CSS to `DraftEditor-root` is not a comprehensive solution because this won't be reflected in the output HTML. 

```.ts
    const fontFamily = getSelectionCustomInlineStyle(newEditorState, ['FONTFAMILY']).FONTFAMILY;
    if (!fontFamily) {
      newEditorState = toggleCustomInlineStyle(newEditorState, 'fontFamily', 'Arial');
    }

    const fontFamily = getSelectionCustomInlineStyle(newEditorState, ['FONTFAMILY',]).FONTFAMILY;
    if (!fontFamily) {
       newEditorState = toggleCustomInlineStyle(newEditorState, 'fontFamily', 'Arial');
    }
```
